packages/tests/src/utils/withGracefulTimeout.ts

```diff
@@ -0,0 +1,38 @@
+import { Logger } from "@waku/utils";
+const log = new Logger("test:mocha-hook");
+
+export function withGracefulTimeout(
+  asyncOperation: () => Promise<void>,
+  timeoutDuration: number,
```

Comment:
let's make it same as `MOCHA_HOOK_MAX_TIMEOUT` global config and optional to pass

Reply:
Yes, makes sense